hooks/tk-multi-publish2/basic/publish_session_geometry.py

```diff
@@ -0,0 +1,574 @@
+﻿# Copyright (c) 2017 Shotgun Software Inc.
+# 
+# CONFIDENTIAL AND PROPRIETARY
+# 
+# This work is provided "AS IS" and subject to the Shotgun Pipeline Toolkit 
+# Source Code License included in this distribution package. See LICENSE.
+# By accessing, using, copying or modifying this work you indicate your 
+# agreement to the Shotgun Pipeline Toolkit Source Code License. All rights 
+# not expressly granted therein are reserved by Shotgun Software Inc.
+
+import os
+import pprint
+import maya.cmds as cmds
+import maya.mel as mel
+import sgtk
+from sgtk.util.filesystem import copy_file
+import traceback
+
+HookBaseClass = sgtk.get_hook_baseclass()
+
+
+class MayaSessionPublishPlugin(HookBaseClass):
```

Comment:
copy/paste: we should change this to something like `MayaSessionGeometryPublishPlugin`.